### PR TITLE
Add QA jobs for GCI M58

### PR DESCRIPTION
**Pull request by yinghan**
Signed-off-by: Ying Han <yinghan@yinghan.mtv.corp.google.com>

@wonderfly Please review.

### CODE REVIEW

**Comment by krzyzacy** [comment]
/assign

Please add an entry similar to https://github.com/kubernetes/test-infra/blob/master/jobs/config.json#L1249-L1255

And fix your CLA? I think you want to use your @google.com email

**Comment by wonderfly** [comment]
Thanks for the pointers @krzyzacy

File: jobs/config.json [hunk 1]
```diff
@@ -1246,6 +1246,14 @@
   ]
 },
 
+"ci-kubernetes-e2e-gce-gci-qa-m58": {
+  "scenario": "kubernetes_e2e",
+  "args": [
+    "--env-file=platforms/gce.env",
+    "--env-file=jobs/ci-kubernetes-e2e-gce-gci-qa-m58.env"
+  ]
+},
```
Comment (krzyzacy) [inline jobs/config.json]: and the slow and serial ones here as well :-)

**Comment by krzyzacy** [comment]
LG for the infra part, @wonderfly double check the configuration :-)

**Comment by yinghan** [comment]
I resent the PR at:
https://github.com/kubernetes/test-infra/pull/2173

Please help to review

**Comment by krzyzacy** [comment]
close in favor of #2173

**Pull request closed without merging**